### DISCUSSION TITLE
Will there be a support for Remix-run

### DISCUSSION THREAD

**Discussion by nemonemi** [nrwl/nx#7794] [Feature Requests]
I'd like to ask if there are any plans to add support for the soon-to-be open-source published Remix-run?

**Comment by brandonroberts** [comment]
We'd have to see how Remix works, but I don't foresee a technical reason why we couldn't support it.

**Reply by always-maap** [reply]
> We'd have to see how Remix works, but I don't foresee a technical reason why we couldn't support it.

Oh, I see in discord. so surely you know v1 is out 😅

**Comment by aridgupta** [comment]
Any idea if/when this is being released?

**Comment by brandonroberts** [comment]
No concrete dates on when, but Victor is looking at working on it soon. Tweet thread for more context https://twitter.com/victorsavkin/status/1468668162786930701?s=20

**Comment by jangerhofer** [comment]
FYI: https://www.npmjs.com/package/@nrwl/remix is in beta!

**Reply by bursteways** [reply]
Remix.run integration would be AMAZING

**Comment by wesleygrimes** [comment]
Just following up on this, any updates as to when `@nrwl/remix` will be updated or moved out of labs? We would like to use it at Jane Tech.

**Reply by nadiTime** [reply]
any news on this?

**Reply by nonameolsson** [reply]
Remix seems to be well used. Any updates on adding support for this?
